lte/gateway/c/oai/tasks/nas/emm/Authentication.c

```diff
@@ -1486,6 +1487,39 @@ static void nas_itti_auth_info_req(
   auth_info_req->visited_plmn  = *visited_plmnP;
   auth_info_req->nb_of_vectors = num_vectorsP;
 
+  /*
+   * Check if we have UE 5G-NR
+   * connection supported in Attach request message.
+   * This is done by checking either en_dc flag in ms network capability or
+   * by checking  dcnr flag in ue network capability.
+   */
+
+#if EMBEDDED_SGW
+  auth_info_req->supportedfeatures.nr_as_secondary_rat = 1;
+  OAILOG_DEBUG(
+      TASK_MME_APP,
+      "S6A_OVER_GRPC is working "
+      "(ue_id = %u)\n",
+      auth_info_req->supportedfeatures.nr_as_secondary_rat);
+
+  OAILOG_DEBUG(
+      TASK_MME_APP,
+      "Value of supportedfeaturesnrassecondaryrat "
+      "(auth_info_req->supportedfeatures.nr_as_secondary_rat = %u)\n",
+      auth_info_req->supportedfeatures.nr_as_secondary_rat);
+#endif
+  ue_mm_context_t* ue_context_p = malloc(sizeof(*ue_context_p));
+  if (ue_context_p->emm_context._ms_network_capability.en_dc) {
+    auth_info_req->supportedfeatures.nr_as_secondary_rat = 1;
+  } else {
+    auth_info_req->supportedfeatures.nr_as_secondary_rat = 0;
+    OAILOG_DEBUG(
+        TASK_MME_APP,
+        "UE is connected on LTE, Dual registration with 5G-NR is disabled for "
+        "(ue_id = %u)\n",
+        ue_context_p->mme_ue_s1ap_id);
+  }
```

Comment:
I don't understand what you are trying to achieve here.

Reply:
Resolved